### PR TITLE
[GH-5951] Update Plugin Configuration Settings > Plugin Management

### PR DESCRIPTION
**Pull request by matthew-src**
#### Summary

This PR updates the Plugin Management section of the Plugins Configuration Settings.

I have general concerns about this section. Parts of it were out of date based on the current plugin-related [documentation](https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/). I was also unable to reference my local test environment, due to what I believe is an issue with the newest docker preview image (see https://github.com/mattermost/mattermost-docker-preview/issues/90).

That said, the settings themselves are simple and should be easy for a technical reviewer to check.

#### Ticket Link

This PR partly addresses https://github.com/mattermost/docs/issues/5951.

### CODE REVIEW

File: source/configure/plugins-configuration-settings.rst [hunk 1]
```diff
@@ -40,146 +40,135 @@ Access the following configuration settings in the System Console by going to **
...
 
-**True**: Enables plugins on your Mattermost server. Use plugins to integrate with third-party systems, extend functionality, or customize the user interface of your Mattermost server. See `documentation <https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta/>`__ to learn more.
-
-**False**: Disables plugins on your Mattermost server.
-
-+---------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"Enable": true`` with options ``true`` and ``false``. |
-+---------------------------------------------------------------------------------------------------+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------+
+| - **true**: **(Default)** Enables plugins on your Mattermost server. See `Use plugins with Mattermost <https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/>`__. | - System Config path: **Plugins > Plugin Management**       |
```
Comment (cwarnermm) [inline source/configure/plugins-configuration-settings.rst:44]: The format for links to other docs pages is generally: "See the `link text here` documentation for details.

File: source/configure/plugins-configuration-settings.rst [hunk 2]
```diff
@@ -40,146 +40,135 @@ Access the following configuration settings in the System Console by going to **
...
+|                                                                                                                                                                                                                            | - Environment variable: ``MM_PLUGINSETTINGS_ENABLEREMOTEMARKETPLACE``        |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------+
+| **Note**: To connect to a remote Marketplace, ``EnableMarketplace`` must be ``true`` and each Mattermost host must have network access to the endpoint set in ``MarketplaceURL``.                                                                                                                         |
```
Comment (cwarnermm) [inline source/configure/plugins-configuration-settings.rst:122]:
```suggestion
| **Note**: To connect to a remote Marketplace, **EnableMarketplace** must be **true** and each Mattermost host must have network access to the endpoint set in **MarketplaceURL**.                                                                                                                         |
```

Comment (cwarnermm) [reply source/configure/plugins-configuration-settings.rst:122]: Historically, the config setting docs focused on the ``config.json`` experience and referenced values ``formatted as code`` to indicate such.

Moving forward, these same docs are more focused on the System Console user experience where referenced values are formatted **boldly** instead. 

It's a small visual difference that can have big impact when used consistently. Please review all applicable instances in this PR and update where needed. Thanks!

File: source/configure/plugins-configuration-settings.rst (hunk 1 above)
Comment (matthew-src) [reply source/configure/plugins-configuration-settings.rst:44]: Got it, I will make that change.

**Comment by matthew-src** [comment]
@cwarnermm, this latest commit includes the changes you suggested, but I left the code format in Installed Plugin State and Plugin Settings unchanged because those descriptions specifically reference how ``config.json`` functions. What do you think I should do with those?

**Comment by cwarnermm** [comment]
Thanks, @matthew-src, and great point! Some config settings are only available via the ``config.json``, and those references should indeed remain formatted to indicate such. You have a great eye for detail and I appreciate your willingness to ask such great questions.

**Pull request merged**